_conferences/flutter-connection-april-2024.md

```diff
@@ -0,0 +1,9 @@
+---
+name: "Flutter Connection"
+website: [https://www.flutterfriends.dev/](https://flutterconnection.io/)
+location: Paris, France
+online: false
+
+date_start: 2024-04-24
+date_end:   2024-04-24
```

Comment:
You need to add a placeholder for `cfp` I guess otherwise CI is unhappy (just put eg 2024-01-01)

Reply:
This sites templates is really not that flexible :D

```suggestion
date_end:   2024-04-24

cfp:
  start: 2024-01-01
  end:   2024-01-01
  site: https://flutterconnection.io/

```